docs/api-docs/provider-apis/tax/overview.mdx

```diff
@@ -30,20 +30,35 @@ Once your tax provider configuration is ready, we'll let you know via email. The
 | Tax provider name | Required | String | Displayed in the active MSF-enabled BigCommerce control panel (e.g. **Settings > Setup > Tax > Add tax service**). | `Sample Tax` |
 | Tax provider type | Required | Production, Sandbox | Hierarchy of tax provider configurations, Production is primary and Sandboxes are secondary, see [sandbox tax provider configuration](#sandbox-tax-provider-configuration) for more information. | `Production` |
 | Partner support email | Required | Email | Used by BigCommerce to contact tax provider, in the case we need to forward a merchant support request. | `support@sampletax.example.com` |
-| Links displayed in an active MSF-enabled control panel | Optional | URL, Title, Description | Link(s) displayed to shoppers when they navigate to **Settings > Setup > Tax > _TaxProviderName_**. One or multiple links supported. | `support.sampletax.com`, `Sample Tax Support`, `The Sample Tax Support website.` |
+| Links displayed in an active MSF-enabled control panel | Optional | URL, Title, Description | Link(s) displayed to merchants when they navigate to **Settings > Setup > Tax > _TaxProviderName_**. One or multiple links supported. | `support.sampletax.com`, `Sample Tax Support`, `The Sample Tax Support website.` |
```

Comment:
Minor question, should this be changed to merchants or **both** merchants and partners? (i.e. do partner stores have the same UI as merchant stores?)

Reply:
I think merchants is fine, as when a partner is using a store they are acting as a merchant. I.e. the fact it is a partner is irrelevant tot he site itself.